docs/en/ingest-management/index.asciidoc

```diff
@@ -2,6 +2,7 @@
 :beats-repo-dir: {beats-root}
 :im-forum: https://discuss.elastic.co/tags/c/elastic-stack/81/stack-ingest-management
 :code-path: {beats-repo-dir}/libbeat/docs/tab-widgets/code
+:elastic-endpoint-integration: Endpoint Security
```

Comment:
@jmikell821 Let me know if you think I should add this to the shared attributes file. I want the attribute name to clearly indicate that it's for the integration as the name appears in Kibana.